rhcos4/profiles/e8.profile

```diff
@@ -82,6 +82,6 @@ selections:
   - sshd_enable_strictmodes
 
   # See also: https://www.cyber.gov.au/ism/guidelines-using-cryptography
-  - var_system_crypto_policy=default
+  - var_system_crypto_policy=default_nosha1
```

Comment:
Only one question: Why was the crypto policy changed from future to default and then default minus sha1?
btw also the link about the rules no longer works

Reply:
Thanks @jhrozek for taking a look at this, I've updated the refs. 

The [ASD crypto guidelines](https://www.cyber.gov.au/acsc/view-all-content/guidance/asd-approved-cryptographic-algorithms) only allow for SHA-2. When we first created the e8 profile for RHEL, the only available policy that prevented SHA-1 being used was 'future'. https://github.com/ComplianceAsCode/content/pull/5024

The 'NO-SHA1' policy module was subsequently added, and the e8 profile was revised. https://github.com/ComplianceAsCode/content/commit/9aa23b0d0dd9632096d97d75e5e5feffb9954001

RHCOS followed a similar pattern - 'NO-SHA1' wasn't available initially, and 'future' was used instead. Now that 'default+NO-SHA1' is available, it's a better fit. 

'default' was an oversight - it should have been 'future' -> 'default:NO-SHA1'. I initially pushed this, and then rebased over the top.

Reply:
Thank you for the detailed reply and fixing the reference.